BigQuery/tests/System/LoadDataAndQueryTest.php

```diff
@@ -158,6 +159,60 @@ public function testRunQuery($useLegacySql)
         }
     }
 
+    public function testInsertRowToTableWithDefaultValueExpression()
+    {
+        // unset default value expression fields so that they are populated automatically
+        unset(
+            $this->row['Name'],
+            $this->row['Age'],
+            $this->row['Weight'],
+            $this->row['IsMagic'],
+            $this->row['CreatedTimestamp']
+        );
+        self::$expectedRows++;
+        $insertResponse = self::$table->insertRow($this->row);
+        sleep(1);
+        $rows = iterator_to_array(self::$table->rows());
+
+        $this->assertTrue($insertResponse->isSuccessful());
+        $this->assertCount(self::$expectedRows, $rows);
+
+        $actualRow = $rows[0];
+
+        $expectedRow = $this->row;
+        $expectedBytes = $expectedRow['Spells'][0]['Icon'];
+        $actualBytes = $actualRow['Spells'][0]['Icon'];
+        unset($expectedRow['Spells'][0]['Icon'], $actualRow['Spells'][0]['Icon']);
+
+        $actualGeography = $actualRow['Location'];
+        unset($expectedRow['Location'], $actualRow['Location']);
+
+        $this->assertEquals('Default Title', $actualRow['Name']);
+        $this->assertEquals(1, $actualRow['Age']);
+        $this->assertEquals(0.5, $actualRow['Weight']);
+        $this->assertEquals(false, $actualRow['IsMagic']);
+
+        $expectedTimestamp = new Timestamp(new \DateTimeImmutable());
+        $actualTimestamp = $actualRow['CreatedTimestamp'];
+        $this->assertEqualsWithDelta(
+            $expectedTimestamp->get()->getTimestamp(),
+            $actualTimestamp->get()->getTimestamp(),
+            10
+        );
+        // unset default value expression fields since they are added automatically
+        unset(
+            $actualRow['Name'],
+            $actualRow['Age'],
+            $actualRow['Weight'],
+            $actualRow['IsMagic'],
+            $actualRow['CreatedTimestamp']
+        );
```

Comment:
Could we leave this out if we just set `$expectedRow` to `$this->row` before we unset everything?

Reply:
We still need to call unset for `$actualRow['CreatedTimestamp']` because that doesn't exist on `$this->row`